core/scm.js

```diff
@@ -72,14 +73,24 @@ const SCHEMA_COMMIT = Joi.object().keys({
         .uri()
         .required()
         .label('Link to commit')
-        .example('https://github.com/scredriver-cd/screwdriver/commit/8843d7f92416211de')
+        .example('https://github.com/screwdriver-cd/screwdriver/commit/8843d7f92416211de')
 }).label('SCM Commit');
 
 const SCHEMA_PR = Joi.object().keys({
     url: Joi.string()
         .uri()
         .label('Link to PR')
-        .example('https://git.ouroath.com/MAILSERVICES/gdpr-event-processor/pull/1')
+        .example('https://github.com/screwdriver-cd/screwdriver/pull/1'),
+    title: Joi.string()
+        .max(512)
+        .label('Title of the pull request'),
+    createTime: Joi.date().iso()
+        .label('Creation Time of the pull request')
+        .example('2018-10-10T21:35:31Z'),
+    username: Joi.string()
+        .max(39)
```

Comment:
Should we change the `user` username field to match? Probably not urgent though
https://github.com/screwdriver-cd/data-schema/blob/master/models/user.js#L14

Reply:
the more i think about it, i should remove this max since the definition of max varies per scm provider